src/watertap_contrib/seto/unit_models/surrogate/vagmd_surrogate.py

```diff
@@ -0,0 +1,1233 @@
+            var_args={
+                # Cooling water volumetric flow rate, fixed to 1265L/h, to maintain vacuum pressure inside the MD module around 200 mbar
+                ("flow_vol_phase", "Liq"): pyunits.convert(
+                    1265 * pyunits.L / pyunits.h, to_units=pyunits.m**3 / pyunits.s
+                ),
+                ("conc_mass_phase_comp", ("Liq", "TDS")): 50,
+                ("temperature", None): 25 + 273.15,
+                ("pressure", None): 101325,
+            },
+            hold_state=True,
+        )
+        self.cooling_out_props[0].temperature.unfix()
+
+        """
+        Add block for the average status in the cooleravg_feed_props
+        """
+        self.avg_cooling_props = (
+            self.config.property_package_seawater.state_block_class(
+                self.flowsheet().config.time,
+                doc="Average properties in the cooler",
+                **tmp_dict,
+            )
+        )
+
+        """
+        Add block for the average status of the feed flow
+        """
+        self.avg_feed_props = self.config.property_package_seawater.state_block_class(
+            self.flowsheet().config.time,
+            doc="Average properties of the feed water",
+            **tmp_dict,
+        )
+
+        tmp_dict["parameters"] = self.config.property_package_water
+        """
+        Add block for the average status in the condenser
+        """
+        self.avg_condenser_props = self.config.property_package_water.state_block_class(
+            self.flowsheet().config.time,
+            doc="Average properties of water in the condenser",
+            **tmp_dict,
+        )
+        self.avg_condenser_props[0].flow_mass_phase_comp["Vap", "H2O"].fix(0)
+        self.avg_condenser_props[0].pressure.fix(101325)
+
+        # Add ports
+        self.add_port(name="feed", block=self.feed_props)
+        self.add_port(name="brine", block=self.evaporator_out_props)
+
+        """
+        Constraint equations
+        """
+        # Cooling system specification
+        if self.config.cooling_system_type == "closed":
+
+            @self.Constraint(doc="Calculate cooling power requirment")
+            def eq_cooling_power_thermal(b):
+                return b.cooling_power_thermal == pyunits.convert(
+                    b.thermal_resistance_hot
+                    * (
+                        b.feed_props[0].temperature
+                        - b.condenser_in_props[0].temperature
+                    ),
+                    to_units=pyunits.kW,
+                )
+
+            @self.Constraint(
+                doc="Calculate inlet cooling water temperature in closed cooling mode"
+            )
+            def eq_cooling_in_temp(b):
+                return (
+                    b.cooling_in_props[0].temperature
+                    == b.feed_props[0].temperature
+                    - pyunits.convert(b.cooling_power_thermal, to_units=pyunits.W)
```

Comment:
Any reason why this is in Watts and kW used in constraint above?

Reply:
The parameter 'thermal resistance' in the equation uses Watts in its unit.